en/sql/schema/table_stmt.rst

```diff
@@ -1606,13 +1607,13 @@ The **alter_table_change_type_strict** parameter determines whether the value co
 
 When the value of the parameter, **alter_table_change_type_strict** is no, it will operate depending on the conditions as follows:
 
-*   Overflow occurred while converting numbers or character strings to Numbers: It is determined based on symbol of the result type. If it is negative value, it is specified as a minimum value or positive value, specified as the maximum value and a warning message for records where overflow occurred is recorded in the log. For strings, it will follow the rules stated above after it is converted to **DOUBLE** type.
+*   Overflow occurred while converting numbers or character strings to Numbers: It is determined based on symbol of the result type. If it is negative value, it is specified as a minimum value or positive value, specified as the maximum value and a warning message for records where overflow occurred is recorded in the log. For strings, it will follow the rules stated above after it is converted to **DOUBLE** type. Overflow also can be returned by the parameter **allow_truncated_string** setting to **no** if the converted string is not to fit the length of the target string type.
```

Comment:
- Overflow also can be --> Overflow can also be 

- if the converted string is not to fit...--> if the converted string does fit...

Reply:
if the converted string is not to fit...--> if the converted string does not fit...

Reply:
Ok, I fixed it.